ladybug_comfort/collection/adaptive.py

```diff
@@ -24,21 +24,47 @@
 class Adaptive(ComfortCollection):
     """Adaptive comfort DataCollection object.
 
+    Args:
+        outdoor_temperature: Either one of the following inputs are acceptable:
+
+            *   1 - A Data Collection of prevailing outdoor temperature values in C.
+                Such a Data Collection must align with the operative_temperature
+                input and bear the PrevailingOutdoorTemperature data type in
+                its header.
+            *   2 - A single prevailing outdoor temperature value in C to be used
+                for all of the operative_temperature inputs below.
+            *   3 - A Data Collection of actual outdoor temperatures recorded over
+                the entire year. This Data Collection must be continouous and
+                must either be an Hourly Collection or Daily Collection. In the event
+                that the input comfort_parameter has a prevailing_temperature_method
+                of 'Monthly', Monthly collections are also acceptable here. Note
+                that, because an annual input is required, this input collection
+                does not have to align with the operative_temperature input.
+
+        operative_temperature: Data Collection of operative temperature (To)
+            values in degrees Celcius.
+        air_speed: Data Collection of air speed values in m/s or a single
+            air_speed value to be used for the whole analysis. If None, this
+            will default to 0.1 m/s.
+        comfort_parameter: Optional AdaptiveParameter object to specify parameters
+            under which conditions are considered acceptable. If None, default will
+            assume ASHRAE-55 criteria.
+
     Properties:
-        prevailing_outdoor_temperature
-        operative_temperature
-        air_speed
-        comfort_parameter
-        neutral_temperature
-        degrees_from_neutral
-        is_comfortable
-        thermal_condition
-        cooling_effect
-        percent_comfortable
-        percent_uncomfortable
-        percent_neutral
-        percent_hot
-        percent_cold
+        *    prevailing_outdoor_temperature
```

Comment:
I noticed that you are using a different number of spaces between the `*` and the text than what you usually do (I think most of the other PRs you sent just have a single space). I don't think it makes a big difference and you don't have to change this whole PR unless it's quick. But it would be nice to be consistent about this going forward.